language/Italian/strings.po

```diff
@@ -1192,1457 +1270,1460 @@ msgstr "Colore opzione menu"
+#: src/frontend/mame/ui/devopt.cpp:252
+#, c-format
+msgid "  Keypad inputs    [%1$d inputs]\n"
+msgstr ""
 
-#: src/frontend/mame/ui/ui.cpp:1097
-msgid "UI:   Disabled"
-msgstr "UI:   Disabilitato"
+#: src/frontend/mame/ui/devopt.cpp:254
+#, c-format
+msgid "  Keyboard inputs    [%1$d inputs]\n"
+msgstr ""
 
-#: src/frontend/mame/ui/ui.cpp:1249
-msgid "Autofire can't be enabled"
-msgstr "Autofire non può essere abilitato"
+#: src/frontend/mame/ui/devopt.cpp:259
+msgid "* Media Options:\n"
+msgstr ""
 
-#: src/frontend/mame/ui/ui.cpp:1302
+#: src/frontend/mame/ui/devopt.cpp:261
 #, c-format
-msgid ""
-"Are you sure you want to quit?\n"
-"\n"
-"Press ''%1$s'' to quit,\n"
-"Press ''%2$s'' to return to emulation."
+msgid "  %1$s    [tag: %2$s]\n"
 msgstr ""
-"Sei sicuro di voler uscire?\n"
-"\n"
-"Premi ''%1$s'' per uscire,\n"
-"Premi ''%2$s'' per contrinuare l'emulazione."
 
-#: src/frontend/mame/ui/ui.cpp:1374
-msgid "Master Volume"
-msgstr "Volume Principale"
+#: src/frontend/mame/ui/devopt.cpp:267
+msgid "* Slot Options:\n"
+msgstr ""
 
-#: src/frontend/mame/ui/ui.cpp:1383
+#: src/frontend/mame/ui/devopt.cpp:274
+msgid "[None]\n"
+msgstr ""
+
+#: src/frontend/mame/ui/auditmenu.cpp:96
 #, c-format
-msgid "%1$s Volume"
-msgstr "%1$s Volume"
+msgid "Audit ROMs for %1$u machines marked unavailable?"
+msgstr "Verifico le ROMs per %1$u le macchine marcate come non disponibili?"
 
-#: src/frontend/mame/ui/ui.cpp:1407
+#: src/frontend/mame/ui/auditmenu.cpp:99
 #, c-format
-msgid "Overclock CPU %1$s"
-msgstr "Overclock CPU %1$s"
+msgid "Audit ROMs for all %1$u machines?"
+msgstr "Verifico le ROMs per tutte %1$u le macchine?"
 
-#: src/frontend/mame/ui/ui.cpp:1416
+#: src/frontend/mame/ui/auditmenu.cpp:104
 #, c-format
-msgid "Overclock %1$s sound"
-msgstr "Overclock %1$s suono"
+msgid "(results will be saved to %1$s)"
+msgstr "(i risultati verranno salvati su %1$s)"
+
+#: src/frontend/mame/ui/auditmenu.cpp:130
+#, c-format
+msgid ""
+"Auditing ROMs for machine %2$u of %3$u...\n"
+"%1$s"
+msgstr ""
+"Verifica delle ROMs per la macchina %2$u di %3$u...\n"
+"%1$s"
+
+#: src/frontend/mame/ui/auditmenu.cpp:142
+msgid "Start Audit"
+msgstr "Avvia Verifica"
+
+#: src/frontend/mame/ui/imgcntrl.cpp:117
+msgid "Cannot save over directory"
+msgstr "Non posso salvare in questa cartella"
+
+#: src/frontend/mame/ui/imgcntrl.cpp:150
+msgid ""
+"The software selected is missing one or more required ROM or CHD images. "
+"Please select a different one."
+msgstr ""
+"Nel software selezionato mancano una o più ROM o immagini CHD richiesti. "
+"Selezionane un'altra."
 
-#: src/frontend/mame/ui/ui.cpp:1437
-#, c-format
-msgid "%1$s Refresh Rate"
-msgstr "%1$s Frequenza di aggiornamento"
+#: src/frontend/mame/ui/info.cpp:31
+msgid "protection"
+msgstr "protezione"
 
-#: src/frontend/mame/ui/ui.cpp:1442
-#, c-format
-msgid "%1$s Brightness"
-msgstr "%1$s Luminosità"
+#: src/frontend/mame/ui/info.cpp:32
+msgid "timing"
+msgstr "temporizzazione"
```

Comment:
"temporizzazione" non si può leggere lol. 
In contesto sarebbe meglio velocità.

Reply:
Attendo la implementazione di queste mdifiche e lo cambio, grazie mille Angelo.

Reply:
Però non ho capito bene cosa voleva dire cuavas...

Reply:
Che non devi rigenerare tutto il file ma solo le stringhe che vuoi cambiare, difatti è abbastanza fastidioso da leggere da parte nostra se vedi il diff.

Reply:
Il mio problema è che non so proprio come procedere correttamente... meglio se rinuncio, ho paura di fare dei casini.